app.js

```diff
@@ -39,11 +39,12 @@ app.use(function(req, res, next) {
   res.setHeader('X-Powered-By', 'Iframely');
 
   res.setHeader('Surrogate-Control', 'max-age=86400, stale-while-revalidate=600, stale-if-error=86400');
+  res.setHeader('Cache-Control', 'public, s-maxage=86400');
```

Comment:
Why is `Cache-Control` set to `public`? its unrelated to `s-maxage`, which is the same as `Surrogate-control`..